docs/topics/setup.rst

```diff
@@ -133,80 +139,6 @@ Optional settings
   cleansed settings in the `django.conf.settings` through the API. Should be
   `False` on production.
 
-Getting a traceback in development
-----------------------------------
-
-There are too many options for this, but it's a commonly asked question.
-
-First off ensure your logs are going somewhere::
-
-    LOGGING = {
-            'loggers': {
-                    'django.request.tastypie': {
-                            'handlers': ['console'],
-                            'level': 'DEBUG',
-                    },
-            },
-    }
```

Comment:
Do you get tracebacks out of the box now? If not, I think some docs would still be helpful.